### PR TITLE
uncontroversial nits from Bob

### PR DESCRIPTION
**Pull request by LPardue**
Takign some of the suggestion on #1802﻿

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -630,7 +630,7 @@ left as an implementation decision.
 
 Absence of a priority parameter in an HTTP response indicates the server's
 disinterest in changing the client-provided value. This is different from the
-logic being defined for the request header field, in which omission of a
+approach for the request header field, in which omission of a
```
Comment (kazuho) [inline draft-ietf-httpbis-priority.md:633]: Take-it-or-leave-it, but maybe: "This is different from that in the request header field".

**Review by kazuho — COMMENTED** [review]
Thank you for the PR.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:633]: or just "This is different from the request header field"

Comment (kazuho) [reply draft-ietf-httpbis-priority.md:633]: That SGTM.

Comment (LPardue) [reply draft-ietf-httpbis-priority.md]: done in #fc5eb16

**Comment by LPardue** [comment]
merging this to clear the remaining issues list out

**Pull request merged**